Optimizely Audience Doc (#5500)
segmentio/segment-docs: src/connections/destinations/catalog/action-optimizely-advanced-audience-targeting/index.md

---
Optimizely Advanced Audience Targeting Destination

> ---

Author: Fixed

{% include content/plan-grid.md name="actions" %}

> [Optimizely Advanced Audience Targeting](https://optimizely.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) allows you to sync your Segment Engage audiences to Optimizely for targeting with Web Experimentation audiences, Feature Experimentation audiences, and CMS Visitor Groups.

Reviewer: ```suggestion
[Optimizely Advanced Audience Targeting](https://optimizely.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} allows you to sync your Segment Engage audiences to Optimizely for targeting with Web Experimentation audiences, Feature Experimentation audiences, and CMS Visitor Groups.
```

This destination is maintained by Optimizely. For any issues with the destination, [contact their Support team](mailto:support@optimizely.com).

{% include content/ajs-upgrade.md %}

## Getting started

**Within your Optimizely Data Platform Account**
1. Navigate to the **App Directory** in the top right (small boxes with + sign icon)
2. Use the autocomplete to find the **Twilio Segment** App
3. Click into the app and click the blue **INSTALL** button in the top left
4. Click the **SETTINGS** tab -> Click **GENERATE** -> and copy the resulting token to your clipboard

**Within your Twilio Segment Account**

1. From the Segment web app, click **Catalog**, then click **Destinations**.

> 2. Find the Destinations Actions item in the left navigation, and click it.

Reviewer 2: ```suggestion
2. Click on the Destinations Tab and search for **Optimizely Advanced Audience Targeting**.
```

Author: Fixed

> 3. Click **Configure Optimizely Advanced Audience Targeting**.

Reviewer 2: ```suggestion
3. Click on the **Configure Optimizely Advanced Audience Targeting** tile.
```

Author: fixed

> 4. Select your **Personas** source as the source (The app will not work with any other source)

Reviewer 2: ```suggestion
4. Select your **Engage Space** as a Source. Note that this Destination will only work when an Engage Space is configured as a Source.
```

Author: Fixed

5. Click **Confirm Source**

> 6. Within the **Settings** tab paste your ODP token into the **API Key** field, select your region, enable the integration and click **Save**

Reviewer 2: ```suggestion
6. Within the **Settings** tab paste your ODP token into the **API Key** field, select your region, enable the integration and click **Save Changes**
```

Author: Fixed

> 7. Click into the **Mappings** section and click **New Mapping**

Reviewer 2: ```suggestion
7. Click into the **Mappings** section, then click **New Mapping**, then click on the **Sync Audience** tile. 
```

Author: Fixed

> 8. In section 3 **Select Mappings** ensure the user identifier you are targeting with your Advanced Audience Targeting integration is mapped to the Optimizely User ID field.

Reviewer 2: ```suggestion
8. In section 3 **Select Mappings** ensure the user identifier you are targeting with your Advanced Audience Targeting integration is mapped to the Optimizely User ID field.
9. Click the Save button to save the Action. 
10. Enable the Action by toggling the Status to **Enabled**.
11. Click back to the **Settings** tab and enable the Destination by toggling the **Enable Destination** toggle to "On", then click "Save Changes". 
```

{% include components/actions-fields.html %}

### **Notes for Using the Destination**

> - Ensure the Advanced Audience Targeting integration is configured in your Optimizely Products so that you can access your connected audiences from Twilio Segment in your Optimizely Product (Web Experimentation, Feature Experimentation, and CMS Visitor Groups). Documentation:

Reviewer 2: Can this be reworded slightly? Maybe break up the sentence?

Author: Fixed

> [Web Experimentation Advanced Audience Targeting](https://docs.developers.optimizely.com/web-experimentation/docs/configure-odp-audience-targeting#integrate-with-other-cdps)

Reviewer 2: Can we explain why we are linking to this location please?

Author: Fixed

> [Web Experimentation Advanced Audience Targeting](https://docs.developers.optimizely.com/web-experimentation/docs/configure-odp-audience-targeting#integrate-with-other-cdps)

Reviewer: ```suggestion
[Web Experimentation Advanced Audience Targeting](https://docs.developers.optimizely.com/web-experimentation/docs/configure-odp-audience-targeting#integrate-with-other-cdps){:target="_blank"}
```

Author: Fixed

> [Feature Experimentation Advanced Audience Targeting](https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting)

Reviewer 2: Can we explain why we are linking to this location please?

Author: Fixed

> [CMS Visitor Groups Advanced Audience Targeting](https://github.com/unrvld/ODP.VisitorGroups)

Reviewer 2: Can we explain why we are linking to this location please?

Author: Fixed

> [CMS Visitor Groups Advanced Audience Targeting](https://github.com/unrvld/ODP.VisitorGroups)

Reviewer: ```suggestion
[CMS Visitor Groups Advanced Audience Targeting](https://github.com/unrvld/ODP.VisitorGroups){:target="_blank"}
```

Author: Fixed

> - If connecting audiences to **Optimizely Web Experimentation** ensure the user id you map to the Optimizely User ID in the destination mappings area is an identifier available on the browser (client side)

Reviewer 2: Can we make this sentence a little clearer to understand please

Author: Fixed
